core/pkg/server/server.go

```diff
@@ -57,6 +62,20 @@ func NewServer(ctx context.Context, addr string, portFile string) (*Server, erro
 	return s, nil
 }
 
+func (s *Server) loopCheckIfParentGone(pid int) bool {
+	for {
+		select {
+		case <-s.ctx.Done():
+			return false
+		case <-time.After(100 * time.Millisecond):
```

Comment:
this value looks like some magic number, maybe make it a const or explain why we choose this rate?

Reply:
done